h2o-py/tests/testdir_jira/pyunit_pubdev_7237_get_actual_auto_parameter.py

```diff
@@ -0,0 +1,240 @@
+import sys
+sys.path.insert(1,"../../")
+
+from tests import pyunit_utils
+import tests
+from h2o.estimators.deeplearning import H2ODeepLearningEstimator
+from h2o.estimators.xgboost import *
+from h2o.estimators.random_forest import H2ORandomForestEstimator
+from h2o.estimators.isolation_forest import H2OIsolationForestEstimator
+from h2o.estimators.aggregator import H2OAggregatorEstimator
+from h2o.estimators.kmeans import H2OKMeansEstimator
+from h2o.estimators.naive_bayes import H2ONaiveBayesEstimator
+from h2o.estimators.gbm import H2OGradientBoostingEstimator
+import numpy as np
+
+
+def test_set_auto_parameters_to_default_values():
+    #testing default setup of following parameters:
```

Comment:
I would split this test into separate tests per algo, will be easier to maintain